src/build_system.ml

```diff
@@ -212,7 +212,21 @@ module File_spec = struct
 end
 
 module Alias0 = struct
-  type t = { dir : Path.t; name : string }
+  include
+    ((struct
+      type t = { dir : Path.t; name : string }
+      let make name ~dir =
+        assert (not (String.contains name '/'));
```

Comment:
We can include this assert in the condition below

Reply:
Done.